### PR TITLE
Dionae Speed Nerf

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Out of concerns of the shoeless slowdown PR. Also, I firmly believe the current dionae speed is an undocumented change, and it conflicts with their durability primarily being balanced around them not being able to catch up to you easily.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

**Comment by Sindorman** [comment]
> Out of concerns of the shoeless slowdown PR. Also, I firmly believe the current dionae speed is an undocumented change, and it conflicts with their durability primarily being balanced around them not being able to catch up to you easily.

It is not an undocumented change. There was PR with requesting the feature of Diona to be able to sprint as people are when they are full on energy and are irradiated. in PR #6337

**Comment by Yonnimer** [comment]
Lore approves of the changes. This shouldn't cause a major difference in speed, but enough of one to help with balancing.

File: code/modules/mob/living/carbon/diona_base.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@ var/list/diona_banned_languages = list(
 	DS.last_location = loc
 	if(light_amount)
 		adjustNutritionLoss(-light_amount) // regenerate our nutrition from light.
-	move_delay_mod = min(initial(move_delay_mod) - light_amount * 1.5, 0)
+	move_delay_mod = min(initial(move_delay_mod) - light_amount * 1, 0)
```
Comment (Sindorman) [inline code/modules/mob/living/carbon/diona_base.dm]: This change actually speeds up diona from light

Comment (ParadoxSpace) [reply code/modules/mob/living/carbon/diona_base.dm]: Yes, I did test it.

Comment (Sindorman) [reply code/modules/mob/living/carbon/diona_base.dm]: Yeah, yours is right. I forgot it is going negative numbers to make it go faster.

**Comment by Sindorman** [comment]
Sorry, I tried to remove suggestion comment

**Pull request merged**